recipes/medcoupling/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "medcoupling" %}
+{% set version = "9.10.0" %}
+
+
+package:
+  name: '{{ name|lower }}'
```

Comment:
```suggestion
  name: medcoupling
```

I think the whole jinja name variable is overused, it leads to too much confusion for no advantage.

Reply:
Yeah, I see your point. Will update